lib/lambda_deployment/configuration.rb

```diff
@@ -44,7 +27,7 @@ def alias_name
     end
 
     def environment
-      @environment ||= calculate_environment_vars
+      @environment_cache ||= @config_env.merge(Dotenv.overload(*Dir.glob('.env*')))
```

Comment:
This will overwrite any variables in the config file with the .env* files. Maybe we should merge the other direction? (e.g. allow the config to override the .env*).

Reply:
Oh, right you are. Fix incoming.